crates/cdk/src/wallet/mint_metadata_cache.rs

```diff
@@ -271,9 +291,11 @@ impl MintMetadataCache {
         &self,
         storage: &Arc<dyn WalletDatabase<Err = database::Error> + Send + Sync>,
         client: &Arc<dyn MintConnector + Send + Sync>,
+        ttl: Option<Duration>,
     ) -> Result<Arc<MintMetadata>, Error> {
         let cached_metadata = self.metadata.load().clone();
         let storage_id = Self::arc_pointer_id(storage);
+        let ttl = ttl.unwrap_or(DEFAULT_TTL);
```

Comment:
Doesn't having this unwrap defeat the purpose of the option? If it is None then we should not auto refresh and should only refresh when the user explicitly calls it.

Reply:
Fixed in 11adbf9f3675f25557047876a6f090b7e3388a0f